beater/burrowbeat.go

```diff
@@ -11,15 +11,15 @@ import (
 	"github.com/elastic/beats/libbeat/beat"
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
-	"github.com/elastic/beats/libbeat/publisher"
+	pub "github.com/elastic/beats/libbeat/publisher"
 
-	"github.com/goomzee/burrowbeat/config"
+	"github.com/Goomzee/burrowbeat/config"
```

Comment:
AFAIK this is not case sensitive, why change this? (this comment applies to all the places you changed this)